lib/negroku/templates/tasks/eye/eye.rb.erb

```diff
@@ -10,8 +10,8 @@ Eye.application '<%= fetch(:application)%>' do
   # except `env`, which merging down
 
   working_dir File.expand_path(File.join(File.dirname(__FILE__), %w[ processes ]))
```

Comment:
maybe we could use  `fetch(:release_path)`. that one doesn't change, and as the idea is the deploy should be idempotent, we need to update the `eye.rb` on every deploy..